js/ui/components/clubhouse.js

```diff
@@ -294,6 +293,10 @@ var ClubhouseComponent = new Lang.Class({
         this._clubhouseButtonManager.destroy();
         this._clubhouseButtonManager = null;
         this._clubhouseSource = null;
+
+        // Reset the AddNotificationAsync old prototype
+        if (this._oldAddNotificationFunc)
+            GtkNotificationDaemon.prototype.AddNotificationAsync = this._oldAddNotificationFunc;
```

Comment:
We should discuss this in the context of the notification disappearing when locking the screen (and not coming back on unlock). Also, if a quest message is shown while the screen is locked, then it'd hit the regular notifications source. So I'd rather remove the resetting of the old function prototype.